software/ente.yml

```
@@ -0,0 +1,11 @@
+name: "Ente"
+website_url: "https://ente.io/"
+source_code_url: "https://github.com/ente-io/ente"
+description: "Fully open source, End to End Encrypted alternative to Google Photos and Apple Photos."
```

The description should describe what the software does.
Additionally, we have a template for mentioning alternative software ([see more info about it here](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md#other-guidelines)).
Possible alternative description; however, feel free to re-write it yourself according to your preferences!
```suggestion
description: "An end-to-end encrypted photo-sharing platform (alternative to Google Photos, Apple Photos)."
```

---

Thanks for the suggestion!